<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_DownloadBooks</class>
 <widget class="QDialog" name="form_DownloadBooks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>175</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>253</green>
        <blue>215</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>253</green>
        <blue>215</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>253</green>
        <blue>215</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>253</green>
        <blue>215</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Объединение баз</string>
  </property>
  <widget class="QComboBox" name="cbBase">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>7</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>EKNEW</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>EKOLD</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RETRO</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>База</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbGetBooks">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>7</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Получить</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbAboutCurrentBase">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>54</y>
     <width>391</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Сводка о базе</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>24</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Всего записей:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Сграблено:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leAmountRecordInBase">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>22</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Осталось:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leGetCount">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="leRemain">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>79</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbLogin">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>login</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbFillBooks">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Отдать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbPrepareBooks">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>8</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Подготовить</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbPreparedFile">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>EKNEW_prepared</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>EKOLD_prepared</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RETRO_prepared</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>33</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Файл</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
